### ISSUE TITLE
How to allow 1 maxselection on a certain section?

### ISSUE THREAD

**Issue by mattmorz** [patosai/tree-multiselect.js#92]
I wanted to behave a section as radio buttons? 
How to I achieve this?

**Comment by patosai** [comment]
The option right now only supports a global setting, so you can't do this. You can modify the code to fit your purposes though:

https://github.com/patosai/tree-multiselect.js/blob/master/src/tree-multiselect/tree.js#L401-L415

This will probably need to be implemented a bit differently.

**Comment by mattmorz** [comment]
Oh ok..is there a way to dynamically uncheck a selection/checkbox?

**Comment by patosai** [comment]
Not natively, but you can get the checkbox element and click it/fire the change event

**Comment by mattmorz** [comment]
Yes, I managed to solve it using that method. But, it's ugly. :)

**Comment by patosai** [comment]
Pull requests welcome :)
